swagger_parser/test/static/static_utils.dart

```diff
@@ -0,0 +1,80 @@
+import 'dart:io';
+import 'package:path/path.dart' as p;
+
+// Create a test project and add all the necessary dependencies
+// to run swagger_parser and the generated clients
+Future<void> setupBaseProject({
+  required String projectPath,
+  required String swaggerParserPath,
+}) async {
+  // Create the test project
+  final createProjectResult =
+      await Process.run('dart', ['create', projectPath, '--force']);
+  assert(
+    createProjectResult.exitCode == 0,
+    'Failed to create project ${createProjectResult.stderr}',
+  );
+
+  // Add swagger_parser to the pubspec.yaml
+  final addSwaggerParseResult = await Process.run(
+    'dart',
+    [
+      'pub',
+      'add',
+      "swagger_parser:{'path':'$swaggerParserPath'}",
+    ],
+    workingDirectory: projectPath,
+  );
+  assert(
+    addSwaggerParseResult.exitCode == 0,
+    'Failed to add swagger_parser dependency ${addSwaggerParseResult.stderr}',
+  );
```

Comment:
Do we need to add a package if we use GenProcessor for generation?

Reply:
Good Point!